docs/extensions/hooks.md

```diff
@@ -146,20 +146,21 @@ export default ({ schedule }) => {
 
 ### Filter Events
 
-| Name                          | Payload              | Meta                                 |
-| ----------------------------- | -------------------- | ------------------------------------ |
-| `request.not_found`           | `false`              | `request`, `response`                |
-| `request.error`               | The request errors   | --                                   |
-| `database.error`              | The database error   | `client`                             |
-| `auth.login`                  | The login payload    | `status`, `user`, `provider`         |
-| `auth.jwt`                    | The auth token       | `status`, `user`, `provider`, `type` |
-| `(<collection>.)items.read`   | The read item        | `collection`                         |
-| `(<collection>.)items.create` | The new item         | `collection`                         |
-| `(<collection>.)items.update` | The updated item     | `keys`, `collection`                 |
-| `(<collection>.)items.delete` | The keys of the item | `collection`                         |
-| `<system-collection>.create`  | The new item         | `collection`                         |
-| `<system-collection>.update`  | The updated item     | `keys`, `collection`                 |
-| `<system-collection>.delete`  | The keys of the item | `collection`                         |
+| Name                          | Payload              | Meta                                    |
+| ----------------------------- | -------------------- | --------------------------------------- |
+| `request.not_found`           | `false`              | `request`, `response`                   |
+| `request.error`               | The request errors   | --                                      |
+| `database.error`              | The database error   | `client`                                |
+| `auth.login`                  | The login payload    | `status`, `user`, `provider`            |
+| `auth.register`               | The new user payload | `identifier`, `provider`, `accessToken` |
```

Comment:
We should add a note that this is only for the oauth2 and openid providers, and that accessToken is an oauth token and not Directus token.